### DISCUSSION TITLE
qBEE自带反吸血功能能完全取代原先的ipfilter.dat吗

### DISCUSSION THREAD

**Discussion by rihalvnium76** [c0re100/qBittorrent-Enhanced-Edition#288] [General]
如题，qBEE开启Auto Ban Unknown Peer from China和Auto Ban Bittorrent Media Player Peer，上传策略设置为反吸血，不另外设置黑白名单，能否实现原先ipfilter.dat的Ban离线下载的效果？

感觉ipfilter.dat还是有用的。
如果qBEE本身的默认设置并不能完全取代ipfilter.dat，那么ipfilter.dat的移除将会是令人遗憾的
![image](https://user-images.githubusercontent.com/57667144/131425345-eeb2d23a-c625-4694-b20e-37d9a0415a70.png)

**Comment by pinyangcong** [comment]
QbitEE自带的可以屏蔽几乎所有离线下载，漏网的只有百度云的go.torrent.dev（-GT****-），要是介意的话可以在C:\Users\{你的用户名}\AppData\Local\qBittorrent这里新建一个peer_blacklist.txt文件
输入

> -GT\d{4}-    .*

保存，重启qbittorrent即可。
注意：最新版qbitEE才支持这项功能
